src/peilbeheerst_model/peilbeheerst_model/crossings_to_ribasim.py

```diff
@@ -2036,13 +2021,10 @@ def WriteResults(self, model, checks):
                 )
 
         if self.model_characteristics["write_goodcloud"]:
-            with open(self.model_characteristics["path_goodcloud_password"]) as file:
-                password = file.read()
+            # Load in password from environment variable
+            password = os.getenv("RIBASIM_NL_CLOUD_PASS")
 
-            cloud_storage = CloudStorage(
-                password=password,
-                data_dir=r"../../../../Ribasim_networks/Waterschappen/",  # + waterschap + '_'+ modelname + '_' + modeltype,
-            )
+            cloud_storage = CloudStorage(password=password, data_dir=os.getenv("RIBASIM_NL_DATA_DIR"))
```

Comment:
The CloudStorage constructor does this for you, so simply using `CloudStorage()` should work.